resolwe/flow/filters.py

```diff
@@ -269,15 +287,38 @@ class Meta(BaseResolweFilter.Meta):
 class CollectionFilter(BaseCollectionFilter):
     """Filter the Collection endpoint."""
 
+    sample_count = filters.NumberFilter(
+        method="count_samples", field_name="entity__count"
```

Comment:
In Resolwe the term `entity` should be used instead of `sample` . So change method `count_samples` to `count_entities`. This applies to the entire PR.